### ISSUE TITLE
Instructions should reflect IDE workflow

### ISSUE THREAD

**Issue by matbalez-zz** [learn-co-curriculum/html-css-html-elements-code-along#1]
At the outset, instead of forking and cloning the instruction should be to simply user the Open button.

The instruction to branch seems unnecessary? Why not just have the user make their code changes in their cloned fork and submit it with learn submit?

@pletcher

**Comment by pletcher** [comment]
Agreed that we should change the instructions to simply using the Open button.

As for branching, I imagine that Grover's thinking was that this way, students would have an easier time reverting their changes if they get themselves into a bind. Having practical understanding of feature branches is something that most students won't have gotten elsewhere -- I kind of think it's worthwhile, even if it is a bit contrived.

But I don't feel super strongly, and we can update the instructions to remove the branches in these code-alongs too.

**Comment by matbalez-zz** [comment]
Seems like Open button change not controversial. 

@jmburges @PeterBell any thoughts on whether we should move away from instructions to branch etc? Strikes me that simple `learn submit` workflow should work (and is what we've told the user to use to date)?

**Comment by matbalez-zz** [comment]
Note @pletcher I think we should also remove instruction to change directory into repository folder (clicking Open button does that automatically for you)

**Comment by PeterBell** [comment]
Open button change not controversial. 

Re: removing the feature branch, I don't think we lose the ability to get out of trouble. It really isn't that much easier to get rid of stuff that is on a branch vs stuff that's on master. I do think there is real value in students learning the "GitHub flow" using feature branches and pull requests, but I don't have an opinion on whether this is the right lab to get people into that or even whether it's necessary as part of this somewhat introductory course. So my vote is an "abstain" - open to either approach.

**Comment by pletcher** [comment]
@PeterBell Fair point, I framed the question the wrong way.

I think I fall slightly in favor of keeping branching if only to get students in the habit of doing so (or at least, to get them to the point where they aren't bewildered if they're asked to do it at some point).

Also, it's helpful for them to get comfortable with the _idea_ of branches, because they'll need to use them for the publishing-to-GitHub project -- and I think that comfort comes easiest with practice.

**Comment by pletcher** [comment]
@matbalez thanks for catching the change directory thing. I've removed that step.

**Comment by matbalez-zz** [comment]
i generally think "more git" is fine for our FSWD students, but keep in mind the audience here; these are going to be relatively non-technical folks who are (generally) not interested in a career as developer. most just want to dabble with HTML/CSS, so i think all this git is largely "too much"

**Comment by pletcher** [comment]
Sorry to have taken so long to get back to this --

I think it's precisely because these students would think of themselves as non-technical that I feel like we can keep the little bit of git that we have in this course. At least to start, I'd rather the curriculum lean towards the challenging side in the hopes that students will surprise themselves with what they're capable of doing -- that's when learning (and Learn) can be really delightful (to use that fuzzy user engagement word).

And if these students consistently find this too hard, we can find ways to guide them a bit more closely through the material.
